src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/events/CreateExceptionRecord.java

```diff
@@ -25,19 +28,50 @@
 
     private final ExceptionRecordMapper mapper;
     private final CcdApi ccdApi;
+    private final List<String> jurisdictionsWithDuplicatePrevention;
 
     public CreateExceptionRecord(
         ExceptionRecordMapper mapper,
-        CcdApi ccdApi
+        CcdApi ccdApi,
+        @Value("${jurisdictions-with-duplicate-er-prevention}") final List<String> jurisdictionsWithDuplicatePrevention
```

Comment:
This config will go away as soon as all services have ElasticSearch enabled in prod.

Reply:
maybe the `-er-` is not needed? it's not present in parameter name or field name

Reply:
This configuration exists already in the app. I'm just adding another reference here. ER is not present in parameter/field names in this class, because the class is all about exception records, so it provides context. The configuration property exists only for the transition period.